corehq/apps/export/const.py

```diff
@@ -51,3 +51,4 @@
 
 FORM_EXPORT = 'form'
 CASE_EXPORT = 'case'
+MAX_EXPORTABLE_ROWS = 1000000
```

Comment:
given an average form could be 10kb (not positive on that number). then that would mean we'd export 10gb of data with one million rows. 10 gb seems a bit high to me.. maybe we should aim for somewhere around 500 - 1000 mb?